dex/src/fees.rs

```diff
@@ -83,61 +75,44 @@ const fn fee_tenth_of_bps(tenth_of_bps: u64) -> U64F64 {
     U64F64(((tenth_of_bps as u128) << 64) / 100_000)
 }
 
-#[inline(always)]
-const fn rebate_tenth_of_bps(tenth_of_bps: u64) -> U64F64 {
-    U64F64(fee_tenth_of_bps(tenth_of_bps).0 + 1)
-}
-
 impl FeeTier {
     #[inline]
-    pub fn from_srm_and_msrm_balances(market: &Pubkey, srm_held: u64, msrm_held: u64) -> FeeTier {
-        let one_srm = 1_000_000;
-
-        if market == &stable_markets::usdt_usdc::ID
-            || market == &stable_markets::msol_sol::ID
-            || market == &stable_markets::ust_usdc::ID
-            || market == &stable_markets::ust_usdt::ID
-            || market == &stable_markets::stsol_sol::ID
-            || market == &stable_markets::usdh_usdc::ID
-        {
+    pub fn from_srm_and_msrm_balances(market: &Pubkey, _srm_held: u64, _msrm_held: u64) -> FeeTier {
+        if market == &flagship_markets::sol_usdc::ID {
+            return FeeTier::Flagship;
+        }
+
+        if market == &stable_markets::usdt_usdc::ID {
             return FeeTier::Stable;
         }
 
-        match () {
-            () if msrm_held >= 1 => FeeTier::MSRM,
-            () if srm_held >= one_srm * 1_000_000 => FeeTier::SRM6,
-            () if srm_held >= one_srm * 100_000 => FeeTier::SRM5,
-            () if srm_held >= one_srm * 10_000 => FeeTier::SRM4,
-            () if srm_held >= one_srm * 1_000 => FeeTier::SRM3,
-            () if srm_held >= one_srm * 100 => FeeTier::SRM2,
-            () => FeeTier::Base,
+        FeeTier::Base
+    }
+
+    fn maker_rate(self) -> U64F64 {
```

Comment:
I've seen this misinterpreted as a maker fee. Rename to `maker_rebate_bps()` and the other to `maker_rebate_qty()`?

Reply:
I wanted to keep it in sync with the taker functions. Will change both in that case